app/basicComponents/Tooltip.js

```diff
@@ -24,27 +30,39 @@ const ToolTipIcon = styled.img`
 const Text = styled.div`
   font-size: 10px;
   line-height: 13px;
-  text-transform: uppercase;
-  color: ${smColors.white};
+  color: ${isDarkModeOn ? smColors.white : smColors.black};
```

Comment:
add text-transform: uppercase; it is important for usage in other components

Reply:
fix